Source/WebCore/animation/CSSPropertyAnimation.cpp

```diff
@@ -2342,6 +2341,22 @@ class PropertyWrapperSVGPaint final : public AnimationPropertyWrapperBase {
     void (RenderStyle::*m_setter)(const StyleColor&);
 };
 
+class PropertyWrapperFontWeight final : public PropertyWrapper<FontSelectionValue> {
+    WTF_MAKE_FAST_ALLOCATED;
+public:
+    PropertyWrapperFontWeight()
+        : PropertyWrapper<FontSelectionValue>(CSSPropertyFontWeight, &RenderStyle::fontWeight, &RenderStyle::setFontWeight)
```

Comment:
I don't think you need the `<FontSelectionValue>` here and that this would be sufficient:

```suggestion
        : PropertyWrapper(CSSPropertyFontWeight, &RenderStyle::fontWeight, &RenderStyle::setFontWeight)
```